### PR TITLE
Replace JUnit 4 with JUnit 5

### PR DESCRIPTION
**Pull request by bennetelli**

### CODE REVIEW

File: src/test/java/com/redhat/cloud/policies/app/AlertHistoryParserTest.java [hunk 1]
```diff
@@ -19,30 +19,31 @@
 import com.fasterxml.jackson.core.JsonProcessingException;
 import com.redhat.cloud.policies.app.model.engine.HistoryItem;
 import com.redhat.cloud.policies.app.rest.PolicyCrudService;
-import org.junit.Assert;
 import org.junit.jupiter.api.Test;
 
 import java.util.ArrayList;
 import java.util.List;
 
+import static org.junit.jupiter.api.Assertions.*;
```
Comment (pilhuhn) [inline src/test/java/com/redhat/cloud/policies/app/AlertHistoryParserTest.java:27]: I think the rule has been not to use star imports

File: src/test/java/com/redhat/cloud/policies/app/OapiTest.java [hunk 2]
```diff
@@ -34,12 +33,14 @@
 import java.nio.file.StandardCopyOption;
 import java.util.Map;
 
+import static org.junit.jupiter.api.Assertions.*;
```
Comment (pilhuhn) [inline src/test/java/com/redhat/cloud/policies/app/OapiTest.java:36]: Star import

Comment (bennetelli) [reply src/test/java/com/redhat/cloud/policies/app/OapiTest.java]: Replaced all star imports with the last commit.

File: src/test/java/com/redhat/cloud/policies/app/RestApiRedirectTest.java [hunk 3]
```diff
@@ -19,14 +19,14 @@
 import static org.hamcrest.CoreMatchers.containsString;
 
 import static io.restassured.RestAssured.given;
+import static org.junit.jupiter.api.Assertions.*;
```
Comment (pilhuhn) [inline src/test/java/com/redhat/cloud/policies/app/RestApiRedirectTest.java]: Star import

File: src/test/java/com/redhat/cloud/policies/app/TestLifecycleManager.java [hunk 4]
```diff
@@ -55,9 +55,6 @@
     @Override
     public void stop() {
         postgreSQLContainer.stop();
-        // Helper to debug mock server issues
```
Comment (pilhuhn) [inline src/test/java/com/redhat/cloud/policies/app/TestLifecycleManager.java:58]: Leave that in please. Those lines are super helpful when the mockserver does not react how you expect it.
One can then uncomment and debug.

I was always thinking to perhaps guard via command-line option, but it's probably overkill

File: src/test/java/com/redhat/cloud/policies/app/UserConfigServiceTest.java [hunk 5]
```diff
@@ -17,14 +17,14 @@
 package com.redhat.cloud.policies.app;
 
 import static io.restassured.RestAssured.given;
+import static org.junit.jupiter.api.Assertions.*;
```
Comment (pilhuhn) [inline src/test/java/com/redhat/cloud/policies/app/UserConfigServiceTest.java]: Star import

File: src/test/java/com/redhat/cloud/policies/app/model/filter/TagsFilterTest.java [hunk 6]
```diff
@@ -19,16 +19,16 @@
 import com.redhat.cloud.policies.app.model.pager.Pager;
 import com.redhat.cloud.policies.app.rest.utils.PagingUtils;
 import org.jboss.resteasy.specimpl.ResteasyUriInfo;
-import org.junit.Assert;
+import org.junit.jupiter.api.Assertions;
```
Comment (pilhuhn) [inline src/test/java/com/redhat/cloud/policies/app/model/filter/TagsFilterTest.java:22]: This and static below?

File: src/test/java/com/redhat/cloud/policies/app/model/filter/TagsFilterTest.java [hunk 7]
```diff
@@ -19,16 +19,16 @@
...
 import org.junit.jupiter.api.Test;
 
 import javax.ws.rs.core.UriInfo;
 import java.net.URI;
 import java.net.URISyntaxException;
 
-import static org.junit.Assert.assertEquals;
+import static org.junit.jupiter.api.Assertions.*;
```
Comment (pilhuhn) [inline src/test/java/com/redhat/cloud/policies/app/model/filter/TagsFilterTest.java]: Star import

**Review by pilhuhn — CHANGES_REQUESTED** [review]
Cosmetics. Otherwise LGTM

File: src/test/java/com/redhat/cloud/policies/app/TestLifecycleManager.java (hunk 4 above)
Comment (bennetelli) [reply src/test/java/com/redhat/cloud/policies/app/TestLifecycleManager.java:58]: added again

**Review by pilhuhn — APPROVED** [review]

**Pull request merged**